src/components/PatchesPage/PatchCard.tsx

```diff
@@ -91,7 +92,9 @@ export const PatchCard: React.VFC<Props> = ({
       status={umbrellaStatus}
       count={count}
       statusCounts={statusCounts}
-      versionId={versionId}
+      href={getVersionRoute(versionId, {
+        statuses: mapUmbrellaStatusToQueryParam[umbrellaStatus],
+      })}
```

Comment:
I was thinking this might be a little odd, though I'm not sure if it's intended behavior:

https://github.com/evergreen-ci/spruce/assets/47064971/c1dea921-587e-4ebb-9d34-31f13fc8df1d

Reply:
Good catch fixed this and added tests